Makefile

```diff
@@ -47,14 +47,14 @@ beta-firefox: release beta-firefox-zip
 .PHONY: beta-firefox
 
 ## dev: Create a debug build for a platform in build/$(browser)/dev.
-## specify browser=(chrome|chrome-mv3|firefox) type=dev
-dev: copy build
+## specify browser=(chrome|chrome-mv3|firefox) type=dev [reloader=1]
+dev: copy build $(BUILD_DIR)/buildtime.txt
 
 .PHONY: dev
 
 ## watch: Create a debug build for a platform in build/$(browser)/dev, and keep
 ##        it up to date as files are changed.
-## specify browser=(chrome|chrome-mv3|firefox) type=dev
+## specify browser=(chrome|chrome-mv3|firefox) type=dev [reloader=1]
```

Comment:
`reloader=1` seems to contradict below, as we're saying here it is optional, but it's also the default. We should instead document that `reloaded=0` is the only option here that makes a difference.

Reply:
Well, the API is that `reloader=1` enables the reloader and `reloader=0` disables it, and that by default it's enabled. As an implementation detail, we don't actually set `reloader=1` because that doesn't do anything.

I didn't add a comment to explain that, since we don't explain browser/type parameters either. But perhaps that was a mistake, I've added a comment to better explain this now. Hopefully that helps clarify?